<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adbprefDialog</class>
 <widget class="QDialog" name="adbprefDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>574</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>452</y>
     <width>229</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="checkButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Check</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Check for update now</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>512</y>
     <width>166</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="minimumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>75</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>122</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Terminal Selection</string>
   </property>
  </widget>
  <widget class="QLabel" name="directoriesLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>272</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Default folders</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Program version</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionLabel_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>macOS</string>
   </property>
  </widget>
  <widget class="QComboBox" name="macTerm">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>183</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>macOS Terminal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>iTerm2 Terminal</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="versionLabel_4">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>747</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Windows</string>
   </property>
  </widget>
  <widget class="QComboBox" name="winTerm">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>740</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>CMD Prompt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Windows Terminal</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="linTerm">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>225</y>
     <width>160</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>160</width>
     <height>32</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Gnome Terminal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XFCE4 Terminal</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>KDE Konsole</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="versionLabel_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>232</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>60</width>
     <height>16</height>
    </size>
   </property>
   <property name="text">
    <string>Linux</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>392</y>
     <width>237</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="backupButton">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default folder for backup sets&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Backup</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="bfilepath">
      <property name="toolTip">
       <string>Pulled files are put in this folder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>349</y>
     <width>237</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="installButton">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default folder for APK files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Install</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ifilepath">
      <property name="toolTip">
       <string>Pulled files are put in this folder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>306</y>
     <width>237</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="downloadButton">
      <property name="minimumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>32</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Default folder for pulled files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string>Pull</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="dfilepath">
      <property name="toolTip">
       <string>Pulled files are put in this folder</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>235</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="versioncheck">
      <property name="text">
       <string>Check for updates at program start</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="scrcpyargs">
      <property name="text">
       <string>Ask for Scrcpy arguments</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="oldfm">
      <property name="text">
       <string>Use old file manager</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="scopecheck">
      <property name="text">
       <string>Check scope on connection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>adbprefDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>adbprefDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>147</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
